packages/medusa/src/utils/remove-undefined-properties.ts

```diff
@@ -0,0 +1,19 @@
+export function removeUndefinedProperties<T extends object>(inputObj: T): T {
+  const removeProperties = (obj: T) => {
+    const res = {} as T
+
+    Object.keys(obj).reduce((acc: T, key: string) => {
+      if (obj[key] !== undefined) {
```

Comment:
We generally used `typeof obj[key] !== 'undefined` in the rest of the codebase
```suggestion
      if (typeof obj[key] !== 'undefined') {
```

Reply:
True, @pKorsholm i though to move the usage of that utility to the transformers pipe, so that we don’t need to remove the undefined in the handler anymore, any counter thoughts on that?

Reply:
I think it's a good idea to move it to the transform pipe, it would definitely remove some of the re-use we otherwise have 😄